src/templates/Table/Table.tsx

```diff
@@ -0,0 +1,57 @@
+import { type ReactNode } from "react";
+import { classNames } from "~/utils";
+import * as tableStyles from "./Table.css";
+
+export const Table = ({
```

Comment:
I think adding a Table should be a first step, but its a very fundamental component, so I think we should have some design unification with the Dashboard itself